master/internal/db/setup.go

```diff
@@ -9,15 +9,20 @@ import (
 
 const maxOpenConns = 48
 
-const cnxTpl = "postgres://%s:%s@%s:%s/%s?sslmode=disable&application_name=determined-master"
+const (
+	cnxTpl         = "postgres://%s:%s@%s:%s/%s?application_name=determined-master"
+	sslTpl         = "&sslmode=%s&sslrootcert=%s"
+	sslModeDisable = "disable"
+)
 
 // Setup connects to the database and run any necessary migrations.
 func Setup(opts *Config) (*PgDB, error) {
 	dbURL := fmt.Sprintf(cnxTpl, opts.User, opts.Password, opts.Host, opts.Port, opts.Name)
+	dbURL += fmt.Sprintf(sslTpl, opts.SSLMode, opts.SSLRootCert)
 	log.Infof("connecting to database %s:%s", opts.Host, opts.Port)
 	db, err := ConnectPostgres(dbURL)
 	if err != nil {
-		return nil, errors.Wrapf(err, "connecting to database: %s", dbURL)
+		return nil, errors.Wrapf(err, "connecting to database: %s:%s", opts.Host, opts.Port)
```

Comment:
potentially on connection failures, we were printing the password 🙃

Reply:
🙃